app/scripts/components/discoveries/hub/featured-discoveries.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import styled from 'styled-components';
+import { discoveries } from 'veda';
+import { VerticalDivider } from '@devseed-ui/toolbar';
+
+import PublishedDate from './pub-date';
+
+import { Card, CardMeta, CardTopicsList } from '$components/common/card';
+import { FoldGrid, FoldHeader, FoldTitle } from '$components/common/fold';
+import {
+  Continuum,
+  ContinuumGridItem,
+  ContinuumCardsDragScrollWrapper,
+  ContinuumDragScroll
+} from '$styles/continuum';
+import { useReactIndianaScrollControl } from '$styles/continuum/use-react-indiana-scroll-controls';
+import { ContinuumScrollIndicator } from '$styles/continuum/continuum-scroll-indicator';
+import { getDiscoveryPath } from '$utils/routes';
+import { Pill } from '$styles/pill';
+
+const allFeaturedDiscoveries = Object.values(discoveries)
+  .map((d) => d!.data)
+  .filter((d) => d.featured);
+
+const FoldFeatured = styled(FoldGrid)`
+  ${FoldHeader} {
+    grid-column: content-start / content-end;
+  }
+`;
+
+export const continuumFoldStartCols = {
+  smallUp: 'content-start',
+  mediumUp: 'content-start',
+  largeUp: 'content-start'
+};
+
+export const continuumFoldSpanCols = {
+  smallUp: 3,
+  mediumUp: 5,
+  largeUp: 8
+};
+
+function FeaturedDiscoveries() {
```

Comment:
This is interesting! Not sure where it got the `type` property (`d.type === "dataset"`) as it doesn't exist.

@nerik I think the tricky thing here would be handling the fact that a discovery has a date and a dataset doesn't (maybe in the future it will have one but won't be called `pubDate`)

Reply:
ideas?

Reply:
I'll try something...